googlespanner/src/main/java/com/yahoo/ycsb/db/GoogleSpannerClient.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2017 YCSB contributors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package com.yahoo.ycsb.db;
```

Comment:
This is something new that we are trying to do. Can you make sure that this is in its own package? com.yahoo.ycsb.db.googlespanner or something similar?

This will be a change in most of the files.